edwin/client/static/js/components/Timeline.js

```diff
@@ -60,6 +60,7 @@ export default class TimelineApp extends ControllerComponent {
     let bugs = this.state.bugs.sortBy((bug) => -bug.get('state').value);
     let undoneBugs = bugs.filter((bug) => bug.get('state') !== BugStates.DONE);
     let doneBugs = bugs.filter((bug) => bug.get('state') === BugStates.DONE);
+    let teamTag = 'edwin-' + this.props.params.team;
```

Comment:
This works, and we can leave it. However, generally for this kind of thing I've been using ES6 template strings. Using that syntax, this line would be 

```
let teamTag = `edwin-${this.props.params.team}`;
```